### PR TITLE
Update CZERTAINLY data

### PR DESCRIPTION
**Pull request by 3keyroman**

### CODE REVIEW

**Comment by vanbroup** [comment]
Is 3Key Company no longer the vendor of CZERTAINLY?

File: README.md [hunk 1]
```diff
@@ -113,11 +113,17 @@ Trident HSM is a general purpose network HSM developed by [I4P](https://www.i4p.
...
 Cryptographic applications are enabled to use Post-Quantum algorithms (SPHINCS+ and Kyber) even in the CC restricted mode that helps with using hybrid solutions and the transition to PQC.
 
-## 3Key Company
+## CZERTAINLY
 
-[CZERTAINLY](https://docs.czertainly.com/docs) is an open-source trust services lifecycle management platform supporting PQC algorithms and initially focusing on the signature schemes Falcon, Dilithium, and SPHINCS+ that are planned to be released as part of the platform in January 2023.
+[CZERTAINLY](https://docs.czertainly.com/docs) is an open-source platform that brings easy and convenient solution to protect baseline security in everyday digital communication by assuring trust lifecycle management and automation consisting of:
+- cryptographic keys
+- digital certificates
+- remote digital signatures
```
Comment (vanbroup) [inline README.md:121]: This description is intended to provide more information about the PQC capabilities of the product.

Can you please update the text accordingly?

**Review by vanbroup — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (3keyroman) [reply README.md:121]: Should we remove this? This is short description of what the CZERTAINLY is and what are the main capabilities, and the second paragraph talks about the PQC algorithms support that can be used with the platform and its capabilities.

If we remove this, then the PQC algorithms will be out of context from my perspective.

Comment (vanbroup) [reply README.md:121]: That might be the best thing, if we start describing everything that is listed we would get a very long page.

Comment (3keyroman) [reply README.md]: See my update...

**Pull request merged**